### ISSUE TITLE
Feature: Plugins - support installing a set of plugins in addition to individually

### ISSUE THREAD

**Issue by mattfarina** [helm/helm#5079]
The documentation for this is in [the proposal](https://github.com/helm/community/blob/master/helm-v3/005-plugins.md#installing-plugins).

The idea is that you can use a command like:

```sh
$ helm init --plugins <file.yaml>
```

with a file (example below) to install a group of plugins:

```
plugins:
- name: helm-template
  url: https://github.com/technosophos/helm-template
- name: helm-value-store
  url: https://github.com/skuid/helm-value-store
- name: helm-diff
  url: https://github.com/databus23/helm-diff
```

Who is interested in working on this?

**Comment by svenvanheugten** [comment]
I'll gladly pick this up.

**Comment by bacongobbler** [comment]
closed in #5109
